src/anndata/__init__.py

```diff
@@ -68,11 +69,13 @@ def __getattr__(attr_name: str) -> Any:
     # Classes
     "AnnData",
     "Raw",
+    "ExtensionNamespace",
```

Comment:
We have a `typing.py` file so you don't have to reexport

Reply:
Forgot to remove it from there. Should the protocol go in `src/anndata/_types.py` or `src/anndata/typing.py`?

Reply:
@flying-sheep can answer this best, but I think `_types.py` actually

Reply:
The names have the following meaning `typing` is for type checking, `types` have runtime checkable things like protocols with `@runtime_checkable` and so on.

And nothing with `_leading_underscore` should ever appear in docs.